src/styles/styles.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "styled-components";
+import "./styles.css";
+
+// Colors
+export const YELLOW = "#ECF955";
+export const GREEN = "#058855";
+export const PINK = "#E8A8CA";
+export const RED = "#DF4E20";
+export const ROYAL_BLUE = "#3C60DD";
+export const LIGHT_BLUE = "#13BCD7";
+export const BLACK = "#170E07";
+export const CREAM = "#FFFAF2";
+
+export const fonts = {
+  cubano: "Cubano",
+  epilogue: "Epilogue",
+  epilogueItalic: "Epilogue Italic",
+  IBMPlexMono: "IBM Plex Mono",
+};
+
+export const H1 = styled.h1`
+  font-family: ${fonts.cubano};
+  text-transform: lowercase;
+  font-size: 60px;
```

Comment:
i totally forgot about this until now EEK but we should use rem instead of px for responsive text sizing

up to you whether you want to address it here or we can do it in a separate PR; all it takes is dividing px by 16